openg2p_registration/models/openg2p_registration.py

```diff
@@ -175,6 +175,44 @@ def _default_company_id(self):
         'registration_id'
     )
 
+    total_quality = fields.Float(
+        string='Total Quality',
+        track_visibility='onchange',
+        required=True,
+    )
+    total_equity = fields.Float(
+        string='Total Equity',
+        track_visibility='onchange',
+        required=True,
+    )
+    retention_progression = fields.Float(
+        string='Retention & Progression',
+        track_visibility='onchange',
+        required=True,
+    )
+
+    grand_total = fields.Float(
+        string='Grand Total',
+        compute='_compute_grand_total',
+        readonly=True,
+        track_visibility='onchange',
+    )
+
+    # @api.multi
+    # def move_stage(self):
+    #     print(self.ids)
+    #     for record in self:
+    #         print(record.id)
+    #         # self.env['openg2p.registration.stage'].browse(record.stage_id)
+    #         pass
+            # record.stage_id =
+
+    @api.depends('retention_progression', 'total_equity', 'total_quality')
+    def _compute_grand_total(self):
+        # print('COMPUTE GRAND TOTAL', self.env['openg2p.registration.stage'].name)
```

Comment:
I can see quite a few commented lines. If they are not required, can we remove them?